src/Microsoft.Framework.HashCodeCombiner.Sources/HashCodeCombiner.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Framework.Internal
+{
+    internal struct HashCodeCombiner
```

Comment:
Sounds like an early optimization. It's a small object that will be cleaned up quickly and easily.